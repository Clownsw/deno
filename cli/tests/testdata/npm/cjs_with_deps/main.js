import chalk from "npm:chalk@4";

console.log(chalk.green("chalk cjs loads"));
