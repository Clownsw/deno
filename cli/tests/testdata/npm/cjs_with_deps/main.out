Download http://localhost:4545/npm/registry/ansi-styles/ansi-styles-4.3.0.tgz
Download http://localhost:4545/npm/registry/chalk/chalk-4.1.2.tgz
Download http://localhost:4545/npm/registry/color-convert/color-convert-2.0.1.tgz
Download http://localhost:4545/npm/registry/color-name/color-name-1.1.4.tgz
Download http://localhost:4545/npm/registry/has-flag/has-flag-4.0.0.tgz
Download http://localhost:4545/npm/registry/supports-color/supports-color-7.2.0.tgz
chalk cjs loads
